usr/share/laptop-mode-tools/modules/runtime-pm

```diff
@@ -125,7 +125,7 @@ if [ x$CONTROL_RUNTIME_AUTOSUSPEND = x1 ] || [ x$ENABLE_AUTO_MODULES = x1 -a x$C
 				if [ x$USE_DEVICE = x1 ]; then
 					if [ -f "$runtime_device"/power/autosuspend_delay_ms ]; then
 						#INFO: Takes value in milliseconds
-						echo $($AUTOSUSPEND_TIMEOUT \* 1000) > "$runtime_device"/power/autosuspend_delay_ms;
+						echo $(($AUTOSUSPEND_TIMEOUT * 1000)) > "$runtime_device"/power/autosuspend_delay_ms 2>/dev/null;
```

Comment:
No. Please don't kill the error message. We'd always want to know the error.

Reply:
The error messages are like this when unplugging:
`/usr/share/laptop-mode-tools/modules/runtime-pm: line 128: echo: write error: Input/output error`
And like this when plugging back in:
`/usr/share/laptop-mode-tools/modules/runtime-pm: line 156: echo: write error: Input/output error`

It's trying to write to every file in `/sys/bus/*/devices/*`. For me, hundreds of those give input/output errors, presumably because of broken drivers. Maybe this should be a configurable option, because personally I don't see the use in seeing any of those errors.

Reply:
IIRC, LMT has/had some code to trap stderr message and pass them to logger with type 'verbose'. Maybe that can be looked at ?

Reply:
Ok, I think I figured it out this time. Thanks for the hint.